frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -409,14 +411,26 @@ export class HtmlDocument extends React.Component {
   handleSaveHtmlDocument = async () => {
     const { state, props } = this
 
+    let newDocumentForApiWithMention
```

Comment:
Maybe this name?
```suggestion
    let contentWithWrappedMentions
```

Reply:
I prefer the name old name, is more specific